cookbooks/aws-parallelcluster-install/resources/stunnel/stunnel_centos7.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+#
+# Copyright:: 2023 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "LICENSE.txt" file accompanying this file.
+# This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, express or implied.
+# See the License for the specific language governing permissions and limitations under the License
+
+provides :stunnel, platform: 'centos' do |node|
+  node['platform_version'].to_i == 7
+end
+unified_mode true
+default_action :setup
+
+use 'partial/_setup'
+
+action_class do
+  def dependencies
+    %w(tcp_wrappers-devel)
+  end
+end
```

Comment:
**Thought:** what about move this package in the list of the *base packages*?

Alternatively remove completely this dependency to have all the OSes aligned? 
Since this feature is not used by `efs-utils` and I do not think that this would be an breaking change since the software should be used only by `efs-utils`, I would keep the installation lighter removing the package.

Reply:
- If they are not needed as base dependencies, I would keep them in the resource/recipes where they are used: this is the approach we have taken for all such dependencies that we use only when installing specific software.
- I would keep the `tcp_wrappers-devel` because it's listed as dependency in the official documentation, despite it maybe not being strictly necessary... I would make only the changes needed to support RHEL8 and not diverge from the official documentation when not strictly needed.